src/status_im/contexts/profile/edit/name/utils_test.cljs

```diff
@@ -0,0 +1,64 @@
+(ns status-im.contexts.profile.edit.name.utils-test
+  (:require
+    [cljs.test :refer-macros [deftest is testing]]
+    [status-im.contexts.profile.edit.name.utils :as utils]
+    [utils.i18n :as i18n]))
+
+(deftest has-emojis-test
+  (testing "it returns true when string has emojis"
+    (is (utils/has-emojis "Hello 😊")))
+  (testing "it returns false when string has no emojis"
+    (is (not (utils/has-emojis "Hello")))))
+
+(deftest has-common-names-test
+  (testing "it returns true when string contains common names"
+    (is (utils/has-common-names "Ethereum")))
+  (testing "it returns false when string does not contain common names"
+    (is (not (utils/has-common-names "Hello")))))
+
+(deftest has-special-characters-test
+  (testing "it returns true when string has special characters"
+    (is (utils/has-special-characters "@name")))
+  (testing "it returns false when string has no special characters"
+    (is (not (utils/has-special-characters "name")))))
+
+(deftest name-too-short-test
+  (testing "it returns true when name is too short"
+    (is (utils/name-too-short "abc")))
+  (testing "it returns false when name is not too short"
+    (is (not (utils/name-too-short "abcdef")))))
+
+(deftest name-too-long-test
+  (testing "it returns true when name is too long"
+    (is (utils/name-too-long (apply str (repeat 25 "a")))))
+  (testing "it returns false when name is not too long"
+    (is (not (utils/name-too-long "abcdef")))))
+
+(deftest validation-name-test
+  (testing "it returns nil when name is nil or empty"
```

Comment:
@mohsen-ghafouri, thank you for covering the code with tests. You might also like the `are` macro to write test examples. This macro is especially nice with short utility-like functions where we need to verify many inputs. We have a few examples in the repo already. Just an FYI, not a suggestion for change :)

Reply:
Thanks @ilmotta ,your suggestion was excellent; now it appears much clearer 🙂